ppmclibs/tinycv_impl.cc

```diff
@@ -41,22 +41,6 @@ using namespace std;
 
 struct Image {
     cv::Mat img;
-    mutable cv::Mat _preped;
-
-    cv::Mat prep() const
-    {
-        if (!_preped.empty() && false)
```

Comment:
I agree. still caching is useful in both cases - as very often we check multiple needles against the scene.